tools/sistr_cmd/sistr_cmd.xml

```diff
@@ -0,0 +1,227 @@
+<tool id="sistr_cmd" name="sistr_cmd" version="0.3.1">
+  <description>
+    Salmonella In Silico Typing Resource commandline tool for serovar prediction
+  </description>
+  <requirements>
+    <requirement type="package" version="0.3.1">sistr_cmd</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="1:" />
+  </stdio>
+  <command><![CDATA[
+  sistr 
+    -f $output_format
+    #if $output_format == "csv"
+      -o sistr-report.csv
+    #elif $output_format == "json"
+      -o sistr-report.json
+    #end if
+    -p $cgmlst_profiles
+    -n $novel_alleles
+    -a $alleles_output
+    -T $tmp_dir
+    #if $keep_tmp
+      -K
+    #end if
+    #if $use_full_cgmlst_db
+      --use-full-cgmlst-db
+    #end if
+    #if $no_cgmlst
+      --no-cgmlst
+    #end if
+    #if $run_mash
+      -m
+    #end if
+    #if $qc
+      --qc
+    #end if
+    #if $threads > 1
+      -t $threads
+    #end if
+    #if $verbosity == 1
```

Comment:
These verbosity level settings are not working for me (no matter what I set it at, it doesn't appear to include `-v` in the command line string).  Maybe it's trying to do integer comparison on strings (or vice-versa)?

Another way to do this is, instead of setting `$verbosity` to a number, set it to `-v` or `-vv` or whatever.  Then just insert `$verbosity` into the command line directly.